distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -45,7 +45,7 @@ type Project_Description
        Arguments:
        - prim_root_file: The primitive root file of the project.
        - prim_config: The primitive config of the project.
-    private Value prim_root_file prim_config
+    private Value prim_root_file ns:Text n:Text
```

Comment:
Something for @Akirathan to review. I replaced usage of `asGuestValue` with a constructor that holds file and two `Text` values in 71cfc0d350617390f4fcb8f57f00443177316989